### ISSUE TITLE
Hyphenization character should probably be ¬

### ISSUE THREAD

**Issue by PonteIneptique** [HN-2021-Boccacius/git-project-Boccace#44]
At the end of each line with hyphenization, you used `'`, maybe it should be `¬`

**Comment by PonteIneptique** [comment]
This is just for your information, it's probably doable to change this automatically.

**Comment by malamatenia** [comment]
You're right, I tried to follow a more true-to-the-manuscript approach but it can actually create conflict with other usages of `'`. With a small check in of the .txt file for the verification corpus, `'`  is also used on two occasions as part of abbreviations (*qd'* for *quod* `Mazarine_Inc59`,` verification_corpus` ` text_Inc59_21-27.txt` line 163 and *istd'* for *istud* respectively at line 173).

This means that since we are sure that `'` equals hyphenation at the end of the line in all instances, it should be replaced by `¬` to avoid redundant noise and inaccuracies should someone wants to analyze the punctuation and signs in the text (that would probably be me in the future so better take care of this now).

The same holds true for the question marks, transcribed unanimously as `:` which creates conflict with the `:`  sign used as a separator between primary and secondary phrases (,).  

Having said this, I will be making all the changes to update the .txt, .XML files, the csv table and will add a clearer explanation to the documentation report by the end of the week.

Thank you for this comment Professor :+1: !!
